src/wwwroot/js/genpage/params.js

```diff
@@ -1155,12 +1155,12 @@ class PromptTabCompleteClass {
         this.registerPrefix('fromto[0.5]', 'Have the prompt change after a given timestep.', (prefix) => {
             return ['\nSpecify in the brackets a timestep like 10 (for step 10) or 0.5 (for halfway through).', '\nIn the data area specify the before and the after separate by "," or "|".', '\nFor example, "<fromto[10]:cat,dog>" switches from "cat" to "dog" at step 10.'];
         });
-        this.registerPrefix('wildcard', 'Select a random line from a wildcard file (presaved list of options)', (prefix) => {
+        this.registerPrefix('wildcard', 'Select a random line from a wildcard file (presaved list of options).', (prefix) => {
             let prefixLow = prefix.toLowerCase();
             return allWildcards.filter(w => w.toLowerCase().includes(prefixLow));
         });
         this.registerAltPrefix('wc', 'wildcard');
-        this.registerPrefix('wildcard[2-4]', 'Select multiple random lines from a wildcard file (presaved list of options) (works same as "random" but for wildcards)', (prefix) => {
+        this.registerPrefix('wildcard[2-4]', 'Select multiple random lines from a wildcard file (presaved list of options) (works same as "random" but for wildcards.)', (prefix) => {
```

Comment:
don't edit unrelated entries here

Reply:
Was changed in the original push to mention the $$variable syntax.  I took that part out but left the period.